images/capi/packer/ova/rockylinux-8.json

```
@@ -11,7 +11,7 @@
   "http_directory": "./packer/ova/linux/{{user `distro_name`}}/http/",
   "iso_checksum": "4eb2ae6b06876205f2209e4504110fe4115b37540c21ecfbbc0ebc11084cb779",
   "iso_checksum_type": "sha256",
-  "iso_url": "https://download.rockylinux.org/pub/rocky/8/isos/x86_64/Rocky-8.5-x86_64-minimal.iso",
+  "iso_url": "https://download.rockylinux.org/pub/rocky/8.5/isos/x86_64/Rocky-8.5-x86_64-minimal.iso",
```

```suggestion
  "iso_url": "https://download.rockylinux.org/pub/rocky/8/isos/x86_64/Rocky-8.6-x86_64-minimal.iso",
```
Agree with @Pearl1594 
the qemu provider also uses the latest version rn which is 8.6